config/initializers/secure_headers.rb

```diff
@@ -59,12 +59,4 @@
   # }
 end
 
-SecureHeaders::Configuration.override(:saml) do |config|
-  provider_attributes = SERVICE_PROVIDERS.values
-
-  acs_urls = provider_attributes.map { |hash| hash['acs_url'] }.compact
-
-  whitelisted_domains = acs_urls.map { |url| url.split('//')[1].split('/')[0] }
-
-  whitelisted_domains.each { |domain| config.csp[:form_action] << domain }
-end
+SecureHeadersWhitelister.new.run
```

Comment:
Doesn't this mean `SecureHeadersWhitelister` will run twice if `FeatureManagement.use_dashboard_service_providers?` returns true since `ServiceProviderUpdater` also calls `SecureHeadersWhitelister.new.run`?